### PR TITLE
Add CMD Ingest to Mission Control Software

### PR DESCRIPTION
**Pull request by epicstar**
## Description

From PlanetaryRobotics/moonranger-mission-control#229, I added some macros to `cmd_ingest_msgs.h`.

## How has this been tested?

See PlanetaryRobotics/moonranger-mission-control#229

## Screenshots (if appropriate):
image
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have read and followed the `CONTRIBUTING.md` file
- [ ] I have applied the .clang-format file to my changes
- [ ] I have commented my code using Doxygen style comments (see Contributing MD for examples), particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the design documentation 
- [ ] I have updated any relevant parameters in the internal ICD
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] If this is a bug-fix, I have added tests that can catch similar bugs in the future
- [x] New and existing unit tests pass locally with my changes
- [x] For cfe apps I have reserved an empty address for my msgs_ids and perf_ids from the [tracking list](https://docs.google.com/spreadsheets/d/1yRUhNTRBOiGLswlsWftNaYlwGKDeNQ_fCnDYFrc_7fU/edit#gid=772812381) and checked that it does not clash with other apps
- [ ] For new added message types, there is a timestamp included via either an explicit timestamp variable in the struct or buried in the a tlmHeader.

### CODE REVIEW

File: fsw/public_inc/cmd_ingest_msgs.h [hunk 1]
```diff
@@ -22,6 +22,7 @@
 */
 #define CMD_INGEST_NOOP_CC 0
 #define CMD_INGEST_RESET_COUNTERS_CC 1
+#define CMD_INGEST_MAX_INGEST 768
```
Comment (epicstar) [inline fsw/public_inc/cmd_ingest_msgs.h:25]: This macro is defined in rover-flight. Please tell me if I should change the rover-flight code, too.

Comment (epicstar) [reply fsw/public_inc/cmd_ingest_msgs.h:25]: @tenzincrouch the other option is defining this as a defined flag via cmake. + is that it probably won't break rover-flight but now people will have to look at CMake to see the defined flags.... Please let me know what you prefer here.

Comment (tenzincrouch) [reply fsw/public_inc/cmd_ingest_msgs.h:25]: I am a fan of moving this all to rover messages. Noting that it may break things short term but can be fixed.

**Review by tenzincrouch — APPROVED** [review]
This is good. Lets make this approach standard where needed.

**Comment by epicstar** [comment]
FYI @tenzincrouch, @srini-cmu, @bdcmu, and @WilliamLeeMoor, this PR is pushed and may possibly break moonranger-rover-flight (see resolved comment).

**Pull request merged**